packages/mdc-menu-surface/foundation.js

```diff
@@ -162,7 +162,7 @@ class MDCMenuSurfaceFoundation extends MDCFoundation {
   }
 
   /**
-   * @param {!MenuSurfaceCorner} corner Default anchor corner alignment of top-left menu surface corner.
+   * @param {!Corner} corner Default anchor corner alignment of top-left menu surface corner.
```

Comment:
I see this one `{!Corner}` but otherwise a whole lot of `{Corner}`s still. What was the verdict on Andy's comment about whether enums require explicit `!`?